### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OdysLam**
user in zendesk reported that he had an issue because he forgot to install the package. It doesn't hurt to over-communicate it.

### CODE REVIEW

File: claim/README.md [hunk 1]
```diff
@@ -293,6 +293,8 @@ You will still be able to see this node in your War Rooms in an **unreachable**
 If you want to reclaim this node into a different Space, you need to create a new identity by adding `-id=$(uuidgen)` to
 the claiming script parameters. For example, using the default claiming script:
```
Comment (joelhans) [inline claim/README.md:294]: I would prefer if we added this sentence in the paragraph above, or below the code block.

```suggestion
the claiming script parameters. Make sure that you have the `uuidgen-runtime` packagen installed, as it is used to run the command `uuidgen` For example, using the default claiming script:
```

Comment (OdysLam) [reply claim/README.md]: done @joelhans

**Review by joelhans — DISMISSED** [review]

File: claim/README.md (hunk 1 above)
Comment (joelhans) [reply claim/README.md]: Thanks! I requested reviews from a few engineers related to claiming.

**Comment by OdysLam** [comment]
@stelfrag @underhood let's try to close this issue.

**Review by underhood — APPROVED** [review]

File: claim/README.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ This node no longer has access to the credentials it was claimed with and cannot
 You will still be able to see this node in your War Rooms in an **unreachable** state.
 
 If you want to reclaim this node into a different Space, you need to create a new identity by adding `-id=$(uuidgen)` to
-the claiming script parameters. For example, using the default claiming script:
+the claiming script parameters. Make sure that you have the `uuidgen-runtime` packagen installed, as it is used to run the command `uuidgen` For example, using the default claiming script:
```
Comment (stelfrag) [inline claim/README.md:294]: Maybe add a dot after `uuidgen` ?

**Comment by joelhans** [comment]
@OdysLam Mind adding the period that @stelfrag suggested? I'll re-approve and help you merge this once that's done.

**Review by stelfrag — APPROVED** [review]

**Review by joelhans — APPROVED** [review]

**Pull request merged**